docs/api-reference/layers/scatterplot-layer.md

```diff
@@ -142,6 +142,13 @@ The maximum line width in pixels. This prop can be used to prevent the path from
 
 If `true`, rendered circles always face the camera. If `false` circles face up (i.e. are parallel with the ground plane).
 
+##### `smoothEdges` (Boolean, optional)
```

Comment:
- Why plural? TextLayer has a font setting called `smoothing`. Or shall we call it `antialiasing` as a commonly understood terminology?
- The documentation should explain why this prop exists and when to disable it.